InnerEye/Azure/azure_runner.py

```diff
@@ -293,14 +295,27 @@ def create_estimator_from_configs(workspace: Workspace, azure_config: AzureConfi
     # create Estimator environment
     framework_version = pytorch_version_from_conda_dependencies(conda_dependencies)
     logging.info(f"PyTorch framework version: {framework_version}")
+
+    if azure_config.node_count > 1:
+        source_config.script_params = {} if source_config.script_params is None else source_config.script_params
+        source_config.script_params.update({'--dist_backend': 'nccl',
+                                            '--init_method': 'tcp://' + '$AZ_BATCH_MASTER_NODE'})
```

Comment:
dist_backend is already set as nccl as default in deep_learning_config so it is True that we can remove that here, but the init method should be 'env://' by default (reading from environment vars) and only tcp' if it's an AML MPI job.

Regarding why I set it here - it's so that it will be passed as an arg to  model_config in runner.parse_and_load_model. Is there a better place to set it?

Reply:
see is_aml_mpi_run (this function is hacky, it relies on the fact that AML updates the init_backend to TCP, whereas for our local runs we are using environment vars